doc/api/buffer.md

```diff
@@ -1290,8 +1306,8 @@ console.log(buf.length);
 ### buf.readDoubleBE(offset[, noAssert])
 ### buf.readDoubleLE(offset[, noAssert])
 
-* `offset` {Number} `0 <= offset <= buf.length - 8`
-* `noAssert` {Boolean} Default: false
+* `offset` {Number} Where to start reading. Must satisfy: `0 <= offset <= buf.length - 8`
```

Comment:
Maybe "Must be within range `[0, buf.length - 8]`". Or maybe the math notation (i.e. difference between `[ ]` and `( )`) would confuse people more.

Also, for consistency we should probably switch these to also be `{Integer}` like other arguments, or `{Uint32}` (mentioned above).